config/nightwatch.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
   test_workers: false,
   test_settings: {
     'default': {
+      globals: {
+        buildtype: process.env.BUILDTYPE
```

Comment:
@annekainicUSDS Who would be the right person / people from core to verify this is golden?